examples/graphql-api/app/utils/index.ts

```diff
@@ -0,0 +1,27 @@
+import { GRAPHQL_API } from "~/config";
+
```

Comment:
/**
 * Clean a GraphQL document to send it via a GET query
 *
 * @param {string} str GraphQL query
 * @returns {string} Cleaned query
 */
const queryCleanner = (str: string): string => str.replace(/([\s,]|#[^\n\r]+)+/g, ' ').trim()